client/src/Signup/Signup.js

```diff
@@ -47,10 +51,11 @@ export function Signup() {
     })
     if (response.status === 201) {
       setSuccess(true)
-    } else {
+    } else if (response.status === 422) {
       const { errors } = await response.json()
-      setErrors(errors[0].detail)
-      // TODO: Sentry
```

Comment:
This comment is removed because we'll be capturing the exceptions from the backend.